src/System.Net.Security/src/System/Net/NegotiateStreamPal.Unix.cs

```diff
@@ -146,8 +250,66 @@ internal static Exception CreateExceptionFromError(SecurityStatusPal statusCode)
             out int newOffset,
             uint sequenceNumber)
         {
-            throw new PlatformNotSupportedException();
+            throw new NotImplementedException("DecryptNtlm: NTLM is not yet supported");
```

Comment:
Until we know what's happening with NTLM, let's make these PlatformNotSupportedExceptions with an appropriate error message from Strings.resx.  If/when NTLM support is added, the exceptions can be removed, but until then, they'll be correct.